doc/browsing_cubes/overlay.ipynb

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
I like this example notebook - it is nice and simple. What I'm not so much a fan of is the blank cell at the bottom of the notebook - is this necessary for a reason that isn't immediately clear or can it be removed?